packages/quantic/scripts/build/util/sfdx.ts

```diff
@@ -12,14 +12,14 @@ export interface SfdxResponse {
 
 /**
  * Executes the given sfdx function and returns the result as an object.
- * @param command The command to be executed, starting with force:.
+ * @param command The command to be executed.
  * @returns {SfdxResponse} The result of the command, if successful.
  * @throws {Error} The error thrown by the sfdx command if unsuccessful.
  */
 export function sfdx<T = SfdxResponse>(command: string): Promise<T> {
```

Comment:
I think it would be better to update the name of this function to a more accurate name now: 
```suggestion
export function sf<T = SfResponse>(command: string): Promise<T> {
```

Reply:
I think it's more confusing personally to just have `sf` instead of `sfdx`.